src/Components/ResultList.tsx

```diff
@@ -59,7 +59,7 @@ export default class ResultList extends React.Component {
         {this.state.results.map((result: Result) => {
```

Comment:
@olamothe, in the slack call, you commented 
`In that sample, the example also does not make a lot of sense… :thinking_face:
Since there’s no condition, it could just simply render “naturally” in the <List>`
I suppose that referred to this list here? What do you mean by render "naturally"? I tried a few things but didn't quite manage to make it work...